pipeline/metadata/flatten_satellite.py

```diff
@@ -251,7 +287,7 @@ def _process_satellite_v1(self, date: str, scan: Any, filepath: str,
         'error': scan.get('error', None),
         'anomaly': not scan['passed'] if 'passed' in scan else None,
         'success': 'error' not in scan,
-        'received': None,
+        'received': [],
```

Comment:
`answer`

Reply:
schema name changes will happen in https://github.com/censoredplanet/censoredplanet-analysis/pull/119